dashboard/app/assets/javascripts/header.js.erb

```diff
@@ -388,3 +389,14 @@ dashboard.header.showProjectHeader = function () {
     location.href = '/p'; <%# TODO: Can't call project_list_path because it's not in scope %>
   });
 };
+
+dashboard.header.updateTimestamp = function () {
```

Comment:
Moved this out of project. Trying to get to a place where project is mostly logic/state, and UI things happen more in header.